app-laptop/msi-keyboard/metadata.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>zohran.londais@gmail.com</email>
+	</maintainer>
+	<longdescription lang="en">
```

Comment:
You'll need to add proxy-maint project as co-maintainer to commit for you. Check example from other proxied ebuilds. Also this longdescription isn't any longer than default ebuild DESCR so it can be removed.